pyspecProcScripts/load_data.py

```diff
@@ -38,31 +38,24 @@ def proc_bruker_deut_IR_withecho_mancyc(s, fl=None):
         fl.next("IR prior to FTing ph")
         fl.image(s.C.setaxis("indirect", "#").set_units("indirect", "scan #"))
     s.ft(
-        ["ph1", "ph2"], unitary=True
+        ["ph1", "ph2"]
     )  # fourier transforming from phase cycle dim to coherence dimension
+    logging.info(
+        strm(
+            "I didn't know what to set the coherence pathway to so you need to manually do this by setting s.set_prop('coherence_pathway') = {'ph1':x, 'ph2':y}"
```

Comment:
this solution is not cool.  You are the one who wrote this module --> do you know what ppg it pertains to?  Why not just paste the main block of the ppg into the docstring of the function, as I ask elsewhere.

Reply:
where else are you asking for this? I don't see - this is processing for a dataset run on the bruker, using AB's old old data.  I honestly don't know which ppg it pertains to if this was for the og 2H IR or what

Reply:
if we don't know which ppg it pertains to, why are we keeping it around?